Pull Request Audio

sandroalvoz opened the pull request:
Added sound reproduction through AudioStreamPlayer. 
Added MainMenuPrueba as an alternative for the MainMenu, enabling the user to set General, Music and SFX volume.

sandroalvoz committed:
fix: problemas addons

sandroalvoz committed:
feat: sfx play test

SFX plays when charging and releasing a dash

sandroalvoz committed:
feat: music with AudioStreamPlayer

sandroalvoz committed:
feat: changed options in MainMenu

Changed options in MainMenu, allowing for the user to modify General, SFX and Music volume using sliders

sandroalvoz committed:
Revert "fix: problemas addons"

This reverts commit 499ad2ac3309a43fa0f7595a24b7e1cc258dbee2.

DakkuaDev committed:
fix: PlayerMovement.gd hotfix

DakkuaDev committed:
Merge branch 'main' into pr/17

DakkuaDev committed:
proj: little changes to files names. test PR scenes.

DakkuaDev commented:
He estado probándola. 

Le he añadido los nuevos cambios y solucionado un par de conflictos que me generaba.

- **Falta que el índice de los idiomas, modifique el idioma de la aplicación.** He visto que tienes una señal conectada a una función que lo comprueba pero por algún motivo no lo está modificando. No he llegado a depurar esa parte.

El resto funciona perfecto. Solucionamos esto y se mergea a la principal.

sandroalvoz committed:
Reapply "fix: problemas addons"

This reverts commit c9419a605221ea35dec61b5fd9a90f2718a92b22.

sandroalvoz committed:
fix: language Selection in OptionsMenu.tscn now works

added localization support for OptionsMenu, CSV file needs to be updated for Master, SFX and Music slider tags

sandroalvoz committed:
proj: updated CSV localization update
